### DISCUSSION TITLE
Error while building a deployment

### DISCUSSION THREAD

**Discussion by romashauk** [vercel/community#4349] [Help]
### Summary

Hi Community, 
We're experiencing an issue with building. 
it says: **Error: Cannot read properties of undefined (reading 'uncompressedSize')**

<img width="1072" alt="Screenshot 2023-10-03 at 20 14 53" src="https://github.com/vercel/community/assets/43916092/abd98e4c-8c87-47ac-8f7b-63457722394f">

Would be great to get any help here

### Steps to Reproduce

<img width="1072" alt="Screenshot 2023-10-03 at 20 14 53" src="https://github.com/vercel/community/assets/43916092/e96a2a2a-e838-4de1-bdd0-49308652e246">

**Comment by amyegan** [comment]
Hey @romashauk! Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help with debugging?
